resources/assets/components/utilities/PageCard/PageCard.js

```diff
@@ -31,7 +34,7 @@ const PageCard = ({ page }) => {
         <img
           alt={showcaseImage.description || `Cover photo for ${showcaseTitle}`}
```

Comment:
I think we'll want a lodash `get` for this one as well to avoid the kaboom. 
```suggestion
          alt={get(showcaseImage, 'description', `Cover photo for ${showcaseTitle}`)}
```